site/docs/guide/deployment-types.md

```diff
@@ -249,7 +249,7 @@ However, there are a number of drawbacks to using this:
    For instance, they indicate that you always receive a response object, so it is your own responsibility to make sure you're not screwing up while using this minor performance optimization.
 
 If you want to use webhook replies, you can specify the `canUseWebhookReply` option in the `client` option of your `BotConfig` ([API reference](https://doc.deno.land/https://deno.land/x/grammy/mod.ts/~/BotConfig)).
-Pass a function that determines whether or not to use webhook reply for the given request, identified by method.
+Pass a function that determines whether or not to use webhook reply for the given request, identified by the method.
```

Comment:
Please revert. It's not about being identified by the method. This is a technical term, and it means _identified by key_, where the key is the method in this case.